pkg/deploy/config.go

```diff
@@ -511,6 +512,12 @@ func (c *Config) CfnParams() ([]types.Parameter, error) {
 			ParameterValue: &p.AnalyticsDeploymentStage,
 		})
 	}
+	if c.Deployment.Parameters.EnableCronHealthCheckInDev != "" {
```

Comment:
This step/function is designed to generate cloudformation Parameters which are used to configure production deployments.

Because this configuration enableCronHealthCheckInDev is intended for development, it is passed into cdk as a flag directly.

You have implemented the cdk flag above correctly which will be used when running `mage deploy:dev`

When we add configuration that is used in both development and and Production deployments, then we would add a cloudformation parameter to the infrastructure stack, and generate a value for it using this method here.

```suggestion
```

Reply:
Thanks for the explainer here 👌